packages/components/src/common/Notification.module.css

```diff
@@ -0,0 +1,34 @@
+.colorBrand {
+  @apply text-brand-700 bg-brand-100;
+  --banner-border-light-color: var(--eds-color-brand-200);
```

Comment:
I think we could remove the `banner-` prefix for these. it's only kind of confusing in the Toast styles though, so not a biggie